middlewares/validators/tasks.js

```diff
@@ -68,7 +68,21 @@ const updateTask = async (req, res, next) => {
   }
 }
 
+const updateSelfTask = async (req, res, next) => {
+  const schema = joi.object().keys({
+    status: joi.string().required()
+  })
+  try {
```

Comment:
Okay I remembered just now that we also need to allow `percentCompleted` to be changed. It has to be a `number`. Please incorporate this small change as well.